storage/client/client.go

```diff
@@ -104,8 +131,14 @@ func (c *StorageClient) Blocks(ctx context.Context, r *BlocksRequest, p *common.
 	return &bs, nil
 }
 
-// Block returns a consensus block.
+// Block returns a consensus block. This endpoint is cached.
```

Comment:
I'd remove this edit, or add the same to the docstring for `Transaction` below too.